client/my-sites/checklist/wpcom-checklist/index.js

```diff
@@ -317,6 +322,23 @@ class WpcomChecklist extends PureComponent {
 						siteSlug={ siteSlug }
 						title={ translate( 'Register a custom domain' ) }
 					/>
+					<TaskComponent
+						bannerImageSrc="/calypso/images/stats/tasks/mobile-app.svg"
+						completed={ this.isComplete( 'mobile_app_installed' ) }
+						completedButtonText={ translate( 'Download' ) }
+						completedTitle={ translate( 'You downloaded the WordPress app' ) }
+						description={ translate(
+							'Download the WordPress app to your mobile device to manage your site and follow your stats on the go.'
+						) }
+						duration={ translate( '%d miniute', '%d minutes', { count: 3, args: [ 3 ] } ) }
```

Comment:
Typo: **miniute**

Reply:
Fixed typo and rebase.
Thanks @ramonjd !